### PR TITLE
README: Add dark/light mode compatibility

### LINKED ISSUE #480: Logo incompatible with dark browser settings
See below:

Capture d’écran 2021-03-22 à 10 28 31

### PR DESCRIPTION
**Pull request by mathieuboudreau**
## Checklist

- [x] I've given this PR a concise, self-descriptive, and meaningful title
- [x] I've linked relevant issues in the PR body
- [x] I've applied [the relevant labels](https://intranet.neuro.polymtl.ca/geek-tips/contributing#pr-labels-a-href-pr-labels-id-pr-labels-a) to this PR
- [ ] I've added relevant tests for my contribution
- [ ] I've updated the documentation and/or added correct docstrings
- [x] I've assigned a reviewer
- [ ] I've consulted [ADS's internal developer documentation](https://github.com/neuropoly/axondeepseg/wiki/Guidelines-for-contribution) to ensure my contribution is in line with any relevant design decisions

## Description

Currently, our logo appeared terrible for users with the dark mode setting,

eg

### Light mode (OLD)

Screenshot 2024-12-19 at 8 23 02 PM

### Dark mode (OLD)

Screenshot 2024-12-19 at 8 23 14 PM

With this PR, the logo will alternate between two PNGs (with transparent backgrounds) for light/dark mode, the only difference between the two being the text color (black/white).

### Light mode (NEW)

Screenshot 2024-12-19 at 8 23 31 PM

### Dark mode (NEW)

Screenshot 2024-12-19 at 8 23 40 PM

I also replaced the U-net figure with a new napari GUI GIF we introduced in our RTD in a recent PR (#850)

## Linked issues

Resolves #480

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
-<img src="https://github.com/neuropoly/axondeepseg/blob/master/docs/source/_static/logo_ads-alpha.png" width="385">
+
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://github.com/axondeepseg/doc-figures/blob/main/logo/logo_ads-dark-alpha.png?raw=true" width="385">
+  <img alt="" src=https://github.com/axondeepseg/doc-figures/blob/main/logo/logo_ads-alpha.png?raw=true" width="385">
```
Comment (joshuacwnewton) [inline README.md:4]:
```suggestion
  <img alt="ADS logo (simplified image of segmented axons/myelin in blue and red beside the text 'AxonDeepSeg')" src=https://github.com/axondeepseg/doc-figures/blob/main/logo/logo_ads-alpha.png?raw=true" width="385">
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,9 @@
...
+</picture>
```
Comment (joshuacwnewton) [inline README.md:5]: This syntax LGTM! The [Art Direction](https://developer.mozilla.org/en-US/docs/Web/HTML/Responsive_images#art_direction) section of the Responsive Images guide explicitly mentions this syntax, and also says that it's fine to remove the "light" case due to the  fallback case. :)

**Review by joshuacwnewton — APPROVED** [review]
Only a small nit. :)

File: README.md (hunk 2 above)
Comment (mathieuboudreau) [reply README.md]: Thanks!!

**Pull request merged**